task-launcher/src/tasks/shared/helpers/getTranslations.js

```diff
@@ -51,7 +51,7 @@ export const getTranslations = async (configLanguage) => {
   async function fetchData() {
     const urls = [
       // This will eventually be split into separate files
-      `https://storage.googleapis.com/road-dashboard/item-bank-translations.csv`,
+      `https://storage.googleapis.com/road-dashboard/item-bank-translations-test4.csv`,
```

Comment:
I will change this for the main one after I finish testing, to not mess up the current version

Reply:
@kachergis this is now :` https://storage.googleapis.com/road-dashboard/item-bank-translations-2024-11-07.csv`